website/docs/docs/dbt-versions/2023-release-notes.md

```diff
@@ -0,0 +1,998 @@
+    - Long node names no longer overflow in the parsed logs section in history drawer
+    - Status pill in history drawer no longer scales with longer command
+    - Tooltip for tab name with a long file name is no longer cut off
+    - Lint button should no longer available in main branch
+
+  </expandable>
+
+- <expandable alt_header='Run history improvements'>
+
+    New usability and design improvements to the **Run History** dashboard in dbt Cloud are now available. These updates allow people to discover the information they need more easily by reducing the number of clicks, surfacing more relevant information, keeping people in flow state, and designing the look and feel to be more intuitive to use.   
+
+    <Lightbox src="/img/docs/release-notes/run-history-improvements.gif" title="Improvements to Run History in dbt Cloud" />
+
+    Highlights include:
+
+    - Usability improvements for CI runs with hyperlinks to the branch, PR, and commit SHA, along with more discoverable temporary schema names
+    - Preview of runs' error messages on hover
+    - Hyperlinks to the environment
+    - Better iconography on run status
+    - Clearer run trigger cause (API, scheduled, pull request, triggered by user)
+    - More details on the schedule time on hover
+    - Run timeout visibility
+
+    dbt Labs is making a change to the metadata retrieval policy for Run History in dbt Cloud. 
+
+    **Beginning June 1, 2023,** developers on the dbt Cloud multi-tenant application will be able to self-serve access to their account’s run history through the dbt Cloud user interface (UI) and API for only 365 days, on a rolling basis. Older run history will be available for download by reaching out to Customer Support. We're seeking to minimize the amount of metadata we store while maximizing application performance. 
+
+    Specifically, all `GET` requests to the dbt Cloud [Runs endpoint](https://docs.getdbt.com/dbt-cloud/api-v2-legacy#tag/Runs) will return information on runs, artifacts, logs, and run steps only for the past 365 days.  Additionally, the run history displayed in the dbt Cloud UI will only show runs for the past 365 days.  
+
+    <Lightbox src="/img/docs/dbt-cloud/rn-run-history.jpg" width="100%" title="The dbt Cloud UI displaying a Run History"/>
+
+    We will retain older run history in cold storage and can make it available to customers who reach out to our Support team. To request older run history info, contact the Support team at [support@getdbt.com](mailto:support@getdbt.com) or use the dbt Cloud application chat by clicking the `?` icon in the dbt Cloud UI. 
+
+  </expandable>
+ 
+- <expandable alt_header='Run details and log improvements'>
+
+    New usability and design improvements to the run details and logs in dbt Cloud are now available. The ability to triage errors in logs is a big benefit of using dbt Cloud's job and scheduler functionality. The updates help make the process of finding the root cause much easier.
```

Comment:
same entry as above - sugg we have two: one for run details and log improvements and the second which is the run history endpoint. instead of combining them.

Reply:
These were in a single note before. I was focused on just moving the release notes over, not enhancing them as they're from last year.